peribolos/knative-extensions-OWNERS_ALIASES

```diff
@@ -61,7 +60,7 @@ aliases:
   - Leo6Leo
   - cali0707
   eventing-kafka-mtsource-approvers:
-  - steven0711dong
+  - pierDipi
```

Comment:
The repository is deperecated and eventing-writers is always an owener so we don't really need anyone in this group